packages/fx-core/src/component/code/spfxTabCode.ts

```diff
@@ -86,10 +86,23 @@ export async function scaffoldSPFx(
     const webpartName = inputs[SPFXQuestionNames.webpart_name] as string;
     const framework = (inputs[SPFXQuestionNames.framework_type] as string) ?? undefined;
     let solutionName: string | undefined = undefined;
+
     if (!isAddSpfx) {
       solutionName =
         ((context as ContextV3).projectSetting?.appName as string) ||
         ((context as PluginContext).projectSettings?.appName as string);
+    } else {
+      const yorcPath = path.join(inputs.projectPath, "SPFx", ".yo-rc.json");
```

Comment:
Persist 'solutionName' in .yo-rc.json if not exist so we can add a webpart to current SPFx project.